src/librustc_feature/active.rs

```diff
@@ -53,6 +53,18 @@ macro_rules! declare_features {
                 $(f(stringify!($feature), self.$feature);)+
             }
         }
+
+        impl std::ops::Index<Symbol> for Features {
+            type Output = bool;
+
+            fn index(&self, feature: Symbol) -> &Self::Output {
```

Comment:
I would prefer a method called `fn enabled(...)` here (as in https://github.com/rust-lang/rust/pull/66945/files#diff-9e39c9410dacf4c689a770a17b8f9813R43) rather than using `features[sym::my_feature]` to make things more obvious.